spec/model/post_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+describe User do
+  # test
+  # first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+  first_post = Post.new(user_id: 1, title: 'Hello', text: 'This is my first post')
+  subject { first_post }
+
+  before { subject.save }
+
+  it 'title should not be blank' do
+    subject.title = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'title length should be between 0 and 250' do
+    subject.title = 'first_post.title.length'
+
+    expect(subject.title.length).to_not be > 250
+  end
+
+  it 'likesCounter must be an integer greater than or equal to 0' do
+    subject.commentsCounter = 1
+    expect(subject.commentsCounter).to be >= 0
+  end
```

Comment:
* Kindly Add unit specs for all of your models' methods and validations as stated in the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.) as you missed tests for `like` and `comment`.